examples/notebooks/robust_models_1.ipynb

```diff
@@ -96,7 +96,7 @@
     "a = 1.339\n",
     "support = np.linspace(-np.pi*a, np.pi*a, 100)\n",
     "andrew = norms.AndrewWave(a=a)\n",
-    "plot_weights(support, andrew.weights, ['$-\\pi*a$', '0', '$\\pi*a$'], [-np.pi*a, 0, np.pi*a]);"
+    "plot_weights(support, andrew.weights, [r'$-\\pi*a$', r'0', r'$\\pi*a$'], [-np.pi*a, 0, np.pi*a]);"
```

Comment:
these line already has double backslashes

Reply:
>>> r'$-\\pi*a$', r'0', r'$\\pi*a$'
('$-\\\\pi*a$', '0', '$\\\\pi*a$')

?

Reply:
How did you you produce this?  i.e. did you copy/paste the line from the .ipynb into the REPL?  Because I'm pretty sure the relevant thing to do is run the .ipynb through the script to convert it to a .py

Reply:
I copied the string `r'$-\\pi*a$', r'0', r'$\\pi*a$'` to an interpreter (spyder python shell).
when I copy it in a regular jupyter notebook, then I get the same.
It is a string that goes to matplotlib, so the only reliable check would be to see how the plot in the notebook looks like.

Reply:
Reverted this edit, and rebased the branch.